Detections/SecurityEvent/NonDCActiveDirectoryReplication.yaml

```diff
@@ -0,0 +1,56 @@
+id: b9d2eebc-5dcb-4888-8165-900db44443ab
+name: Non Domain Controller Active Directory Replication
+description: |
+  'This query uses Windows Event ID 4662 in order to detect potential attempts to synz an active directory object leveraging directory replication services (DRS)
```

Comment:
...potential attempts to sync not synz. ;)

Reply:
fixed